homeassistant/components/arcam_fmj/strings.json

```diff
@@ -1,8 +1,5 @@
 {
     "config": {
-        "title": "Arcam FMJ",
-        "step": {},
-        "error": {},
-        "abort": {}
+        "title": "Arcam FMJ"
```

Comment:
we should remove this whole file, there is no config flow.

Reply:
Here it is:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/arcam_fmj/config_flow.py

Reply:
Then the manifest is wrong: https://github.com/home-assistant/home-assistant/blob/ea5d3ce85a9fc29a6425c63fbec8f61099d43149/homeassistant/components/arcam_fmj/manifest.json#L4

Reply:
Oh I guess it's correct. It just only works with import and not with user facing config flow. my bad.

Reply:
So, it's right ?

Reply:
Can we resolve this @balloob ?